test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```

Comment:
- use double quote `" "` in Rails

Khi đọc tut thì sẽ bỏ qua phần test em nhé.
Vì có file này trong pull nên tiện thể anh sẽ comment luôn :D